content/realtime/channels/channel-parameters/deltas.textile

```diff
@@ -0,0 +1,207 @@
+
+h4(#delta-example-sse). Delta example with SSE
+
+You can subscribe to messages in delta mode, using the SSE transport, as follows.
+
+```
+(() => {
+    const key = '{{API_KEY}}';
+    const channel = 'sample-app-sse';
+    const url = `https://realtime.ably.io/event-stream?channels=${channel}&v=1.1&key=${key}&delta=vcdiff`;
+    const eventSource = new EventSource(url);
+    const channelDecoder = new DeltaCodec.CheckedVcdiffDecoder();
+
+    eventSource.onmessage = (event) => {
+        /* event.data is JSON-encoded Ably Message (see https://www.ably.io/documentation/realtime/types#message) */
+        const message = JSON.parse(event.data);
+        const { id, extras } = message;
+        let { data } = message;
+
+        try {
+            if (extras && extras.delta) {
+                data = channelDecoder.applyBase64Delta(data, id, extras.delta.from).asUtf8String();
+            } else {
+                channelDecoder.setBase(data, id);
+            }
+        } catch(e) {
+            /* Delta decoder error */
+            console.log(e);
+        }
+
+        /* Process decoded data */
+        console.log(data);
+    };
+})();
+```
+
+h4(#delta-example-unenv-sse). Delta example with unenveloped SSE
+
+```(() => {
+    const key = '{{API_KEY}}';
+    const channel = 'sample-app-sse';
+    const url = `https://realtime.ably.io/event-stream?channels=${channel}&v=1.1&key=${key}&delta=vcdiff&enveloped=false`;
+    const eventSource = new EventSource(url);
+    const channelDecoder = new DeltaCodec.VcdiffDecoder();
+
+    eventSource.onmessage = (event) => {
+        let data = event.data;
+
+        try {
+            if (DeltaCodec.VcdiffDecoder.isBase64Delta(data)) {
+                data = channelDecoder.applyBase64Delta(data).asUtf8String();
+            } else {
+                channelDecoder.setBase(data);
+            }
+        } catch(e) {
+            /* Delta decoder error */
+            console.log(e);
+        }
+
+        /* Process decoded data */
+        console.log(data);
+    };
+})();
+```
+
+h4(#delta-example-mqtt). Delta example with MQTT
+
+```
+const mqtt = require('mqtt');
+const { VcdiffDecoder } = require('./lib');
+
+const options = {
+  keepalive: 30,
+  username: 'FIRST_HALF_OF_API_KEY',
+  password: 'SECOND_HALF_OF_API_KEY',
+  port: 8883
+};
+const client = mqtt.connect('mqtts:mqtt.ably.io', options);
+const channelName = 'sample-app-mqtt';
+const channelDecoder = new VcdiffDecoder();
+
+client.on('message', (_, payload) => {
+  let data = payload;
+
+  try {
+    if (VcdiffDecoder.isDelta(data)) {
+      data = channelDecoder.applyDelta(data).asUint8Array();
+    } else {
+      channelDecoder.setBase(data);
+    }
+  } catch(e) {
+    /* Delta decoder error */
+    console.log(e);
+  }
+
+  /* Process decoded data */
+  console.log(data);
+});
+
+client.subscribe(`[?delta=vcdiff]${channelName}`);
+```
```

Comment:
I am not entirely sure, but shouldn't this section have reference documentation for the methods etc?

Reply:
@tomczoink do you know how we set this up?

Reply:
tom has addressed this in a commit